### ISSUE TITLE
Check scheduled cloud adapters and autodelay

### ISSUE THREAD

**Issue by mcm1957** [ioBroker/ioBroker.repochecker#224]
Adapters of type CLOUD and SCHEDULED should NOT set seconds in cron spec as this will disable autmatic distribution.

**Comment by Apollon77** [comment]
... unless the have good reasons for it that should be checked

**Comment by mcm1957** [comment]
Yes, of course only a warning one can ignore if there is a good reason.
AND if too many good reasons exist, we can / will adapt the check :-)
